common/djangoapps/static_replace/models.py

```diff
@@ -27,3 +27,27 @@ def __repr__(self):
 
     def __unicode__(self):
         return unicode(repr(self))
+
+
+class AssetExcludedExtensionsConfig(ConfigurationModel):
+    """Configuration for the the excluded file extensions when canonicalizing static asset paths."""
+
+    class Meta(object):
+        app_label = 'static_replace'
+
+    excluded_extensions = TextField(
+        default='html',
+        help_text='The file extensions to exclude from canonicalization.  No leading period required. ' +
+        'Values should be space separated i.e. "html svg css"'
+    )
+
+    @classmethod
```

Comment:
Making this a `classmethod` might cause confusion down the road, since you can call class methods on instances. You could imagine a case where you have a reference to a `AssetExcludedExtensionsConfig` that is not the current one. In that case, calling `old_config.get_excluded_extensions()` would ignore the extensions specified for that object, and instead grab the current active. Please make this a regular instance method and have the caller get the `.current()`.